modules/claims_api/spec/lib/claims_api/bd_spec.rb

```diff
@@ -7,7 +7,9 @@
   subject { described_class.new }
 
   let(:ews) do
-    create(:claims_api_evidence_waiver_submission, :with_full_headers_jesse, claim_id: '60897890')
+    create(:claims_api_evidence_waiver_submission, :with_full_headers_jesse, claim_id: '60897890',
+                                                                             id: '43fc03ab-86df-4386-977b-4e5b87f0817f',
+                                                                             tracked_items: [234, 235])
```

Comment:
Same as above (`tracked_item_ids`?)

Reply:
@tycol7 [tracked_items]( https://github.com/department-of-veterans-affairs/vets-api/blob/master/db/schema.rb#L379) is the attribute in the database